test/items.js

```diff
@@ -0,0 +1,226 @@
+		it('Should add an Item', async () => {
+			let event = await createAndSaveEvent(exampleEvent);
+
+			return await chai
+				.request(app)
+				.post('/items/add')
+				.set('content-type', 'multipart/form-data')
+				.field('name', 'Boba')
+				.field('price', 4.99)
+				.field('eventId', event._id.toString())
+				.attach(
+					'img',
+					fs.readFileSync(`${__dirname}/assets/sample-image.jpg`),
+					'sample-image.jpg'
+				)
+				.then(async (res) => {
+					assert.strictEqual(res.status, 200);
+					let items = await Item.find();
+					assert.strictEqual(items.length, 1);
+
+					await deleteImageFromS3(res.body.imgKey);
+				});
+		});
+
+		it('Should reject creation of an Item without image file', async () => {
+			let event = await createAndSaveEvent(exampleEvent);
+
+			return await chai
+				.request(app)
+				.post('/items/add')
+				.set('content-type', 'multipart/form-data')
+				.field('name', 'Boba')
+				.field('price', 4.99)
+				.field('eventId', event._id.toString())
+				.then(async (res) => {
+					assert.notStrictEqual(res.status, 200);
+				});
+		});
+
+		it('Should reject creation of an Item without name, price, or eventId', async () => {
+			return await chai
+				.request(app)
+				.post('/items/add')
+				.set('content-type', 'multipart/form-data')
+				.attach(
+					'img',
+					fs.readFileSync(`${__dirname}/assets/sample-image.jpg`),
+					'sample-image.jpg'
+				)
+				.then(async (res) => {
+					assert.notStrictEqual(res.status, 200);
+				});
+		});
+	});
+
+	describe('put /items/weight', () => {
+		it('Should update weight of an item', async () => {
+			let event = await createAndSaveEvent(exampleEvent);
+			let savedItem = await createAndSaveItem(exampleItemA, event._id);
+
+			return await chai
+				.request(app)
+				.put('/items/weight')
+				.send({
+					itemId: savedItem._id.toString(),
+					weight: 12.7,
+				})
+				.then(async (res) => {
+					assert.strictEqual(res.status, 200);
+					let item = await Item.findById(savedItem._id.toString());
+					assert.strictEqual(item.weight, 12.7);
+					await deleteImageFromS3(savedItem.imgKey);
+				});
+		});
+
+		it('Should reject update weight of an item without id/weight', async () => {
+			let event = await createAndSaveEvent(exampleEvent);
+			let savedItem = await createAndSaveItem(exampleItemA, event._id);
+
+			return await chai
+				.request(app)
+				.put('/items/weight')
+				.then(async (res) => {
+					assert.strictEqual(res.status, 400);
+					await deleteImageFromS3(savedItem.imgKey);
+				});
+		});
+
+		it('Should return error status if Item not found', async () => {
+			let event = await createAndSaveEvent(exampleEvent);
+			let savedItem = await createAndSaveItem(exampleItemA, event._id);
+
+			return await chai
+				.request(app)
+				.put('/items/weight')
+				.send({
+					itemId: 'clearlyWrongId',
+					weight: 12.7,
+				})
+				.then(async (res) => {
+					assert.notStrictEqual(res.status, 200);
```

Comment:
I think all resource not found statuses are 404, so you could say `strictEqual(res.status, 404)` here